src/Appwrite/Specification/Format.php

```diff
@@ -56,14 +38,6 @@ abstract class Format
         'license.url' => '',
     ];
 
-    /**
-     * @param App $app
-     * @param array $services
-     * @param Route[] $routes
-     * @param Model[] $models
-     * @param array $keys
-     * @param int $authCount
-     */
     public function __construct(App $app, array $services, array $routes, array $models, array $keys, int $authCount)
```

Comment:
Did we decide to remove doc comments in favor of type definition?

Reply:
I don't see too much value for doc types anymore when not needed.

It just creates a lot of noise in the code and increases the code we have to write, without giving any value.